.azure/pipelines/benchmarks.yml

```diff
@@ -1,8 +1,5 @@
 # This configuration builds the repository and runs benchmarks
 
-# Don't run CI for this config
-trigger: none
```

Comment:
The unfortunate consequence of this will be that this build definition will run for all pushes to master, not just PRs.

You might try this instead:

```yml
trigger: none

pr:
- '*'
```

Reply:
It shouldn't because we disabled it from the UI. It worked as expected before adding this line. But I can try.